fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -2179,6 +2177,13 @@ ACTOR void setupAndRun(std::string dataFolder,
 		testConfig.storageEngineExcludeTypes.push_back(4);
 	}
 
+	// TODO: Currently backup and restore related simulation tests are failing when run with rocksDB storage engine
```

Comment:
I created a radar to fix backup and restore related simulation tests. Wanted to enable joshua for rocksdb so that we can get some test coverage while I fix backup related tests. That is the reason I excluded the backup related tests for now. Will target to fix them by next sprint.